MbDotNet/Models/Predicates/InjectPredicate.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Newtonsoft.Json;
+
+namespace MbDotNet.Models.Predicates
+{
+    public class InjectPredicate : PredicateBase
+    {
+        [JsonProperty("inject")]
+        public string JavaScriptFunction { get; private set; }
```

Comment:
I think I prefer the name `InjectedFunction` for this property and the constructor argument.